### PR TITLE
Add setting for read only profile fields BB-5316

### PR DESCRIPTION
**Pull request by rollick**

### CODE REVIEW

File: bluebottle/settings/base.py [hunk 1]
```diff
@@ -290,6 +290,8 @@
     'django.core.context_processors.static',
     'django.core.context_processors.tz',
     'django.contrib.messages.context_processors.messages',
+    'tenant_extras.context_processors.conf_settings',
```
Comment (eodolphi) [inline bluebottle/settings/base.py]: I cannot find that context processor anywhere

File: bluebottle/utils/context_processors.py [hunk 2]
```diff
@@ -1,8 +1,33 @@
...
+
+
+def tenant_properties(request):
+    extra_properties = tenant_extra_properties(request)
+    settings = json.loads(extra_properties['settings'])
+
+    # If tenant has SAML enabled then we also return a list
+    # of read-only user profile properties.
+    try:
+        mappings = properties.TOKEN_AUTH['assertion_mapping']
```
Comment (eodolphi) [inline bluebottle/utils/context_processors.py]: Maybe the name should indicate that this is to set the read only properties.

Can't this be decoupled slightly more from tenant_extra_properties a little by this just doing the read only setting?

Comment (rollick) [inline bluebottle/utils/context_processors.py]: It is extending the "settings" property in the context. I wanted to just move the context processor out of tenant-extras packages but thought we can do that later.

File: bluebottle/settings/base.py (hunk 1 above)
Comment (rollick) [inline bluebottle/settings/base.py]: It is in the tenant-extras package.

File: bluebottle/utils/context_processors.py (hunk 2 above)
Comment (eodolphi) [inline bluebottle/utils/context_processors.py]: Yeah I saw that it was extending it, I just thought that we could decouple this a little more. If we'll move the context processor out, then my comment above is not relevant.

File: bluebottle/settings/base.py (hunk 1 above)
Comment (eodolphi) [inline bluebottle/settings/base.py]: I mis-typed. Never mind.

**Pull request merged**